src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -1906,7 +1906,7 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
             if (string.IsNullOrEmpty(incomingString))
                 throw new ArgumentException("value string should not be empty.");
 
-            var numberOfPoints = incomingString.Count(x => x == Configurations.CategoryDelimiter);
+            var numberOfPoints = incomingString.Count(x => x == Configurations.CategoryDelimiterString[0]);
```

Comment:
Perhaps this way, so that the indexer is only called once:

```csharp
var c = Configurations.CategoryDelimiterString[0];
var numberOfPoints = incomingString.Count(x => x == c);
```